cluster/saltbase/salt/top.sls

```diff
@@ -56,6 +56,14 @@ base:
     - docker
     - kubelet
 {% endif %}
+{% if pillar.get('enable_node_logging', '').lower() == 'true' %}
+  {% if pillar['logging_destination'] is defined and pillar['logging_destination'] == 'elasticsearch' %}
```

Comment:
Any reason not to lift the first condition up into line 59, and just make this an if/elif block? Just a minor nit